domain/src/main/java/no/difi/meldingsutveksling/receipt/GenericReceiptStatus.java

```diff
@@ -0,0 +1,26 @@
+package no.difi.meldingsutveksling.receipt;
+
+import javax.xml.bind.annotation.XmlEnum;
+import javax.xml.bind.annotation.XmlEnumValue;
+import javax.xml.bind.annotation.XmlType;
+
+@XmlType
+@XmlEnum(String.class)
+public enum GenericReceiptStatus implements ReceiptStatus {
+    @XmlEnumValue("SENT") SENT("Sent"),
+    @XmlEnumValue("DELIVERED")DELIVERED("Delivered"),
+    @XmlEnumValue("READ") READ("Read"),
+    @XmlEnumValue("FAIL") FAIL("Fail"),
+    @XmlEnumValue("OTHER") OTHER("Other");
+
+    private final String status;
+
+    GenericReceiptStatus(String status) {
+        this.status = status;
+    }
+
+    public String getStatus() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1161)